week1_gp6_zad2.cpp

```diff
@@ -0,0 +1,40 @@
+#include <iostream>
+using namespace std;
+unsigned int SIZE=100;
+
+struct OrderedPair{
+    int first;
+    int second;
+};
+
+void PrintOrderedPair(const OrderedPair& PairToPrint){
+    cout<<"("<<PairToPrint.first<<"; "<<PairToPrint.second<<")"<<endl;
+}
+
+bool isFirstBigger(const OrderedPair& FirstPair, const OrderedPair& SecondPair){
+    if(FirstPair.first>SecondPair.first || (FirstPair.first==SecondPair.first && FirstPair.second>SecondPair.second))
+        return true;
+    return false;
+}
+
+OrderedPair findBiggestOne(OrderedPair* arr, size_t SIZE){
+        int minIndex=0;
+        OrderedPair minimal;
```

Comment:
Having this variable is rather redundant. I'd say it's better to use `arr[minIndex]` instead. What confuses me though is that the name `findBiggestOne` implies I'm not gonna be changing the original array (`arr`) but I'm only going to be searching in it for a `specific` element. Considering this it's not a good idea to have those changes (lines 32 & 33) or to rename this one to `sort` :))